Filters/exceptions.txt

```diff
@@ -46,6 +46,10 @@
 ! https://github.com/AdguardTeam/AdGuardSDNSFilter/issues/707
 @@||health.halodoc.com^|
 !
+! Blocked by CNAME ||customer.io^
+! https://github.com/AdguardTeam/AdguardFilters/issues/99102
+@@||email2.123formbuilder.com^|
```

Comment:
I guess this `@@||email*.123formbuilder.com^|` will be ok in case, if they use other subdomains.